storage/migrations/03_statecheck_snapshots.up.sql

```diff
@@ -1,5 +1,7 @@
 BEGIN;
 
+-- TODO: Not needed anymore. The state check is now manually managing the snapshot schemas.
```

Comment:
Let's comment out all the SQL in this file then, WDYT? It will make it more obvious what the expected current state is.

The uber proper way would be to introduce a new migration file that deletes the schema, but I think that's overly bureaucratic for the current stage of maturity.

Reply:
Updated